packages/ui-components/src/components/Modal/Modal.test.tsx

```diff
@@ -314,14 +316,14 @@ describe("Modal", () => {
     )
     const user = userEvent.setup()
     expect(screen.getByRole("dialog")).toBeInTheDocument()
-    waitFor(() => {
-      user.keyboard("{Tab}")
+    const _p = waitFor(async () => {
```

Comment:
I delete the variable and tests were green.

Reply:
Deleted too, but added lint ignore rule